collector/config/config.go

```diff
@@ -16,19 +16,23 @@ const (
 	DefaultValidatorRegistryAddress = "terra1aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
 	DefaultAirDropRegistryContract  = "terra_dummy_airdrop"
 	DefaultRewardDispatcherContract = "terra_dummy_rewarddispatcher"
+
+	DefaultUpdateDataInterval = 30
```

Comment:
You can set this as a duration right away, btw

Reply:
done